### ISSUE TITLE
ERROR on parsing `HkTypeParamClause ‘=>>’ Type`.

### ISSUE THREAD

**Issue by KaranAhlawat** [tree-sitter/tree-sitter-scala#310]
### Commit of tree-sitter-scala you tested this on

a5c4c29

### A code sample showing the error

```scala
class X extends Y[
  [A <: B] =>> C,
] {}
```

### Show the error node

```
(class_definition class body: (identifier)
 (extends_clause extends
  arguments: 
   (generic_type type: (type_identifier)
    type_arguments: 
     (type_arguments [
      (ERROR [)
      (infix_type left: (type_identifier) operator: (operator_identifier)
                  right: (type_identifier))
      (ERROR ] = > >)
      ])))
 (template_body { }))
```

### Where are you experiencing this error?

treesit (GNU Emacs 29+)

**Comment by susliko** [comment]
@KaranAhlawat Thanks for reporting!

**Comment by KaranAhlawat** [comment]
Fixed in #312. Closing.
